vspace/src/main/webapp/WEB-INF/views/staff/dashboard/dashboard.html

```diff
@@ -1,9 +1,42 @@
 <html layout:decorate="~{layouts/main_staff}">
 <head>
-	<script>
+	<script th:inline="javascript">
+		var authenticatedName = [[${authenticationName}]]
+		var spaceLinksSize = [[${spaceLinksSize}]]
 		$(function () {
 		  $('[data-toggle="tooltip"]').tooltip()
 		});
+		
+	function closeSpaceLinkAlert(){
+		$("#spaceLinkAlert").hide();
+	}
+	
+	function close(){
+		$("#error").hide();
+	}
+	
+	function deleteSpaceLinks(){
+		var url = "[(@{'/staff/dashboard/deleteSpaceLinksWithNullSource?'+${_csrf.parameterName}+'='+${_csrf.token}})]";
+		$.ajax({
+    		url: url,
+    		type: 'DELETE',    
+    		success: function(data) {
+    			//put succesfully deleted 
```

Comment:
?

Reply:
Added success message, should have put that in my todo, my bad